src/main/java/org/folio/fqm/service/EntityTypeService.java

```diff
@@ -70,7 +77,54 @@ public Optional<EntityType> getEntityTypeDefinition(UUID entityTypeId) {
    */
   @Transactional(readOnly = true)
   public ColumnValues getColumnValues(UUID entityTypeId, String columnName, @Nullable String searchText) {
-    String fql = String.format(COLUMN_VALUE_SEARCH_FQL, columnName, searchText == null ? "" : searchText);
+    searchText = searchText == null ? "" : searchText;
+
+    EntityType entityType = entityTypeRepository.getEntityTypeDefinition(entityTypeId)
+      .orElseThrow(() -> new EntityTypeNotFoundException(entityTypeId));
+    EntityTypeColumn column = entityType.getColumns()
+      .stream()
+      .filter(col -> Objects.equals(columnName, col.getName()))
+      .findFirst()
+      .orElseThrow(() -> new ColumnNotFoundException(entityType.getName(), columnName));
```

Comment:
This new stuff at the top of the method is what broke a bunch of tests. Before, it didn't matter if tests used a random UUID as the entity type ID. But now that it actually looks it up, there needs to be an `EntityType` that backs the UUID